global.json

```diff
@@ -15,6 +15,7 @@
   },
   "msbuild-sdks": {
     "Microsoft.DotNet.Arcade.Sdk": "9.0.0-beta.24080.3",
-    "Microsoft.DotNet.Helix.Sdk": "9.0.0-beta.24080.3"
+    "Microsoft.DotNet.Helix.Sdk": "9.0.0-beta.24080.3",
+    "Microsoft.Build.NoTargets": "3.7.0"
```

Comment:
Where is the NoTargets msbuild sdk uses in this repo?

Reply:
https://github.com/dotnet/sdk/blob/b12388169f6fafc63ae6926e6470a9d706fc680a/src/Containers/packaging/package.csproj#L1

Reply:
Got it. Shouldn't this PR also remove that explicit version there?

Reply:
Good call - I was thinking of that as well - will do.

Reply:
Pushed the update to this PR with a fix for this.

Reply:
Thanks. Please do the same in the other PRs as well.

Reply:
Sure - will update `razor` PR soon: https://github.com/dotnet/razor/pull/9886